homeassistant/components/openweathermap/weather.py

```diff
@@ -91,6 +104,8 @@ def __init__(
             manufacturer=MANUFACTURER,
             name=DEFAULT_NAME,
         )
+        self.mode = mode
+        self.weather_coordinator = weather_coordinator
```

Comment:
The coordinator is already accesible via `self.coordinator`

Reply:
Removed, and later call to `self.weather_coordinator` changed to `self.coordinator`